src/Application.vala

```diff
@@ -201,7 +201,11 @@ public class Terminal.Application : Gtk.Application {
         );
 
         var new_window_action = new SimpleAction ("new-window", null);
-        new_window_action.activate.connect (new_window);
+        new_window_action.activate.connect (() => {
+            var win = new MainWindow (this, false); // Do not restore tabs
+            win.add_tab_with_working_directory (null);  // Add default tab
+            win.present ();
+        });
```

Comment:
please, keep the separate callback, it makes easier to breakpoint it during debug after.

Reply:
Done